### ISSUE TITLE
Add recommended show page issue

### ISSUE THREAD

**Issue by duramato** [pymedusa/Medusa#1606]
image 1:

![image](https://lol.doyou.bid/Bmz7l.png)

image 2:
![image](https://lol.doyou.bid/tcJCm.png)

irc quote:
```
when i open the page it show's me this "image 1"
if i resize the page it shows the remaining shows
like for example when opening the browser console "image 2"
tbn between those 2 screenies i didn't refreshed the page or anything, only pressed F12 and as soon as it showed up the shows showed
<p0psicles> Yeah i got the same bug. Really strange. Must be some weird js or vue bug. Assign it to omg.
```

@OmgImAlexis

**Comment by OmgImAlexis** [comment]
@p0psicles looks like isotope isn't firing on pageload, right? If you press F12 it'll resize the DOM's window and cause a page update so that's probably what's causing this.

**Comment by p0psicles** [comment]
@OmgImAlexis , the $('[v-cloak]').removeAttr('v-cloak'); is interfering. As soon as it removes that, it's showing the first (and single) poster.

**Comment by p0psicles** [comment]
Should fix #1606

**Comment by OmgImAlexis** [comment]
@p0psicles Close this then?
